resources/pg_user.rb

```diff
@@ -13,3 +13,15 @@
 attribute :superuser,
 :kind_of => [TrueClass, FalseClass],
 :default => false
+
+attribute :admin_username,
```

Comment:
I don't see the need to have `admin_` on these. Should probably just `username` and `password`.